lib/Doctrine/Persistence/ObjectRepository.php

```diff
@@ -34,14 +36,17 @@ public function findAll();
      *
      * @param mixed[]       $criteria
      * @param string[]|null $orderBy
-     * @param int|null      $limit
-     * @param int|null      $offset
      *
      * @return object[] The objects.
      *
      * @throws UnexpectedValueException
      */
-    public function findBy(array $criteria, ?array $orderBy = null, $limit = null, $offset = null);
+    public function findBy(
+        array $criteria,
+        ?array $orderBy = null,
+        ?int $limit = null,
+        ?int $offset = null
+    ) : array;
```

Comment:
```suggestion
    ) : iterable;
```

Reply:
I would prefer to make this change in another PR. I want to maintain behavior in this PR as much as possible unless it is out of our control when adding existing documented types.